bin/check-http.rb

```diff
@@ -204,9 +207,12 @@ class CheckHttp < Sensu::Plugin::Check::CLI
          description: 'Check the response contains at least BYTES bytes',
          proc: proc(&:to_i)
 
-  option :response_code,
+  option :response_codes,
```

Comment:
As mentioned in other comments this constitutes a breaking change and I think its avoidable.